src/app/widgets/reactive-fields/reactive-label/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './reactive-label.module';
```

Comment:
**Major:**Why do we need this?
We have the module that we can export.

Reply:
When you import this module in another module

- with `index.ts`: `import { ReactiveLabelComponent } from '@widgets/reactive-label';`
- without `index.ts`: `import { ReactiveLabelComponent } from '@widgets/reactive-label/reactive-label.module';`

write less -> makes the code clearer and easier